include/dlaf/eigensolver/backtransformation/mc.h

```diff
@@ -149,10 +151,13 @@ void BackTransformation<Backend::MC, Device::CPU, T>::call_FC(
       }
     }
 
+    const GlobalElementIndex v_offset(v_start.row() * mb, v_start.col() * mb);
```

Comment:
Just checking my understanding of the algorithm here: are the blocks assumed to be square in the current implementation, or can the shape be freely chosen for this part and it's chosen to be square?

Reply:
Yes, exactly. This algorithm assumes square blocksize for `mat_v`. Assumptions valid for the algorithm, and not just for a specific implementation, are enforced by the dispatcher.

https://github.com/eth-cscs/DLA-Future/blob/416aa0e9bf871c2acb3c0b806f4ff9500d09e2ec/include/dlaf/eigensolver/backtransformation.h#L39-L44